### PR TITLE
Add dropdown to user cause trigger

### PR DESCRIPTION
**Pull request by NotMyFault**
As opposed to other parts, there's no dropdown next to the build trigger status, if the build was triggered by a user.

The change proposed adds one :)

### Proposed changelog entries

* Add a dropdown menu to the build trigger, if it's instantiated by a user.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by timja — COMMENTED** [review]
when would you want this? it makes sense you can click through to the user to find out more about them, but why would you want to go to their view or credentials from there?

File: core/src/main/resources/hudson/model/Cause/UserCause/description_zh_TW.properties
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/Cause/UserCause/description_zh_TW.properties:23]: This one is (probably, it looks like it but I cannot read it) a nice example that splitting out the user mention from the localized string doesn't always work because grammar.

Wouldn't just adding the `class` into each localized string do it?

File: core/src/main/resources/hudson/model/Cause/UserIdCause/description_hu.properties
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/Cause/UserIdCause/description_hu.properties:4]: As above.

File: core/src/main/resources/hudson/model/Cause/UserIdCause/description_ko.properties
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/Cause/UserIdCause/description_ko.properties:24]: As above.

File: core/src/main/resources/hudson/model/Cause/UserIdCause/description_zh_TW.properties
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/Cause/UserIdCause/description_zh_TW.properties:23]: As above.

File: core/src/main/resources/hudson/model/Cause/UserCause/description_ko.properties
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/Cause/UserCause/description_ko.properties:23]: As above.

**Review by daniel-beck — DISMISSED** [review]
Likely breaking several localizations?

File: core/src/main/resources/hudson/model/Cause/UserCause/description_zh_TW.properties
Comment (NotMyFault) [reply core/src/main/resources/hudson/model/Cause/UserCause/description_zh_TW.properties:23]: Yeah, that works out too.

**Review by timja — APPROVED** [review]
sure, I still don't think this is needed but it's used elsewhere like this

**Review by basil — APPROVED** [review]
Sure, why not. :smile:

---

This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Pull request merged**
